tests/CMakeLists.txt

```diff
@@ -36,18 +36,20 @@ foreach(src IN LISTS libsrc)
     list(APPEND test_srcs "../${src}")
 endforeach()
 add_library(testobj OBJECT ${test_srcs})
+# NOTE: this is needed to import definitions and include path from libnetconf2::netconf2
+target_link_libraries(testobj libnetconf2::netconf2)
```

Comment:
Why is this testobj lib used?
Note: All sources are already contained in libnetconf2!

Reply:
I believe the tests use some private functions/definitions, that is the reason it is linked directly.